practical-07/src/components/NavBar.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <nav class="fixed-top">
+    <div class="nav-bar">
+    <div>
+      <router-link to="/" class="home-btn nav-link">Home</router-link>
+    </div>
+    <div class="nav-heading" v-if="!['LoginForm', 'RegisterForm'].includes($route.name)">
+      <span>{{ heading }}</span>
+    </div>
+    <div>
+      <router-link to="/LoginForm" class="login-btn nav-link" v-if="!['LoginForm', 'RegisterForm'].includes($route.name)">Login</router-link>
```

Comment:
Codacy has a fix for the issue: [Replace `······<router-link·to="/LoginForm"·class="login-btn·nav-link"·v-if="!['LoginForm',·'RegisterForm'].includes($route.name)">Login</router-link` with `········<router-link⏎··········to="/LoginForm"⏎··········class="login-btn·nav-link"⏎··········v-if="!['LoginForm',·'RegisterForm'].includes($route.name)"⏎··········>Login</router-link⏎········`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
        <router-link
```